src/components/Trade/hooks/useSwapper/useSwapper.ts

```diff
@@ -114,10 +114,7 @@ export const useSwapper = () => {
   )
 
   const feeAsset = useAppSelector(state =>
-    selectFeeAssetById(
-      state,
-      sellTradeAsset?.asset ? sellTradeAsset.asset.assetId : 'eip155:1/slip44:60',
-    ),
+    selectFeeAssetById(state, sellTradeAsset?.asset.assetId ?? 'eip155:1/slip44:60'),
```

Comment:
Using default swap sell asset as currently defined by `useSwapper.getDefaultPair()`. I wasn't sure if I would be able to make use of the hook method in conjunction with the `useAppSelector` since I don't think we can guarantee the `useSwapper` hook value exists before the `useAppSelector` hook executes and to my knowledge, can wrap `useAppSelector` in another hook. Correct me if I am wrong or clean way to get default without the hardcode.

Reply:
this is ok, until @elmutt / @DaoDev44 implement a default `supportedChainIds` or similar in a swapper, in which case we can safely default to a known good fee asset from a swapper statically.